### PR TITLE
Weather Forecast Capstone Project Final Review

### PR DESCRIPTION
**Pull request by raihan2bd**
# Weather Forecast
Hi there 👋 ,
In this PR:
- Use React documentation.
- Use React components.
- Use React props.
- Use React Router.
- Connect React and Redux.
- Handle events in a React app.
- Write integration tests with a React testing library.
- Use styles in a React app.
- Use React life cycle methods.
- Apply React best practices and language style guides in code.
- Use store, actions, and reducers in React.
- Fix the Linter's errors
- Add necessary styles and also add a professional readme file.

#### Api Links
[REST COUNTRIES](https://restcountries.com/#api-endpoints-v2-all) 
[Open Weather](https://openweathermap.org/api)
#### Design guideline link
[Nelson Sakwa](https://www.behance.net/gallery/31579789/Ballhead-App-%28Free-PSDs%29)

***If you have any questions about my PR feel free to ask. 👍 Thanks for visiting my PR 🙏 .***

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+
+- **User can see list of countries by selecting the region**
+- **User can see a the weather of a country**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> ## Project demo
+![Capture](https://user-images.githubusercontent.com/35267447/217789912-ead664b3-086f-4de2-8df8-e0229f60ae8a.PNG)
```
Comment (mikethreels) [inline README.md]: - [ ] the application should be deployed using Heroku, Netlify or GitHub Pages. unfortunately, I'm not able to find a link to the deployed project.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi @raihan2bd, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- linters are passing

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- [ ] [OPTIONAL] It would be a very nice addition if the countries would be sorted in alphabetical order.
- [ ] [OPTIONAL] It would also be nice if the user could just click on the card to go to the details page instead of having to click on the arrow.
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (raihan2bd) [reply README.md]: Hi, @mikethreels ✋  Thanks a lot for your valuable feedback. 👍 I will implement your suggestions before the next review. Thanks once again 🙏

**Review by TadesseDev — APPROVED** [review]
### Hi @raihan2bd 
### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy (1)

## Highlights
 - [ ] Good job making the changes requested by the previous code reviewers. ✅
 - [ ]  Good commit history. ✅
 - [ ]  Good PR title/description. ✅
 - [ ]  All linters passed ✅
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by raihan2bd** [comment]
@mikethreels, @Tadesse-Alemayehu  Thanks a lot for reviewing my project👍

**Pull request merged**